nomad-aws/security_groups.tf

```diff
@@ -1,16 +1,12 @@
+data "aws_vpc" "current" {
```

Comment:
What does this change do? I looked around a bit, but couldn't find it referenced anywhere

Reply:
Nothing, it was a remnant of my earlier approach. Thank you for spotting!